src/aflak_primitives/src/lib.rs

```diff
@@ -630,6 +590,14 @@ fn run_average(im: &WcsArray3, start: i64, end: i64) -> Result<IOValue, IOErr> {
     Ok(IOValue::Image2d(wrap_with_unit))
 }
 
+fn run_integral(im: &WcsArray3, start: i64, end: i64) -> Result<IOValue, IOErr> {
+    reduce_array3_slice(im, start, end, |slices| slices.sum_axis(Axis(0)))
+}
```

Comment:
```rust
if start == end {
         return Err(IOErr::UnexpectedInput(format!(
             "division by zero will occur because {} = {}",
             start, end
         )));
}
```

現状で、上記のcondition checkの適切なエラーメッセジは`division by zero will occur`ではないかも。
プログラミングがそもそもsliceが作れなくて、`image_val.slice(s![start..end, .., ..])`のときにpanicすると思う。
`Expect start < end, however got start = {} and end = {}`,というのはどう？

Reply:
そうですね、これの次のコードでキャッチできているので、この部分は削除します。